src/components/Header.tsx

```
@@ -1,24 +1,76 @@
+import Tab from '@mui/material/Tab'
+import { Link, useLocation } from '@tanstack/react-router'
+import { HardDrive, Settings } from 'lucide-react'
+import ModeToggle from '@/components/ModeToggle'
+
+interface IconProps {
+  size?: number
+}
+
+function DockerIcon({ size = 18 }: IconProps) {
+  return (
+    <svg viewBox="-1 70 514 372" width={size} height={size} fill="currentColor" className="mr-2">
```

Biome (2.4.4)

[error] Alternative text title element cannot be empty

(lint/a11y/noSvgWithoutTitle)